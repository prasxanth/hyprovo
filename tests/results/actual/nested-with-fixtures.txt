2021-11-16 09:27:35.309 - INFO - Pass ✓... (test-add-variable): add pass
2021-11-16 09:27:35.311 - ERROR - Fail ✗... (test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-16 09:27:35.315 - INFO - Pass ✓... (test-sub-variable): sub pass
2021-11-16 09:27:35.317 - ERROR - Fail ✗... (test-sub-variable): sub fail 
                                    Expected: 4
                                    Got: 0
2021-11-16 09:27:35.321 - INFO - Pass ✓... (test-add-variable): add pass
2021-11-16 09:27:35.323 - ERROR - Fail ✗... (test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-16 09:27:35.327 - INFO - Pass ✓... (test-sub-variable): sub pass
2021-11-16 09:27:35.329 - ERROR - Fail ✗... (test-sub-variable): sub fail 
                                    Expected: 4
                                    Got: 0
2021-11-16 09:27:35.337 - INFO - Pass ✓... (test-add-variable): add pass
2021-11-16 09:27:35.341 - ERROR - Fail ✗... (test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-16 09:27:35.349 - INFO - Pass ✓... (test-sub-variable): sub pass
2021-11-16 09:27:35.352 - ERROR - Fail ✗... (test-sub-variable): sub fail 
                                    Expected: 4
                                    Got: 0
2021-11-16 09:27:35.358 - INFO - Pass ✓... (test-add-variable): add pass
2021-11-16 09:27:35.361 - ERROR - Fail ✗... (test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-16 09:27:35.366 - INFO - Pass ✓... (test-sub-variable): sub pass
2021-11-16 09:27:35.368 - ERROR - Fail ✗... (test-sub-variable): sub fail 
                                    Expected: 4
                                    Got: 0
