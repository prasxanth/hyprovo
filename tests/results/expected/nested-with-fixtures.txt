2021-11-14 11:26:49.735 - INFO - Pass ✓... (test-add-variable): add pass
2021-11-14 11:26:49.739 - ERROR - Fail ✗... (test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-14 11:26:49.748 - INFO - Pass ✓... (test-sub-variable): sub pass
2021-11-14 11:26:49.754 - ERROR - Fail ✗... (test-sub-variable): sub fail 
                                    Expected: 4
                                    Got: 0
2021-11-14 11:26:49.763 - INFO - Pass ✓... (test-add-variable): add pass
2021-11-14 11:26:49.769 - ERROR - Fail ✗... (test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-14 11:26:49.778 - INFO - Pass ✓... (test-sub-variable): sub pass
2021-11-14 11:26:49.782 - ERROR - Fail ✗... (test-sub-variable): sub fail 
                                    Expected: 4
                                    Got: 0
2021-11-14 11:26:49.795 - INFO - Pass ✓... (test-add-variable): add pass
2021-11-14 11:26:49.799 - ERROR - Fail ✗... (test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-14 11:26:49.808 - INFO - Pass ✓... (test-sub-variable): sub pass
2021-11-14 11:26:49.812 - ERROR - Fail ✗... (test-sub-variable): sub fail 
                                    Expected: 4
                                    Got: 0
2021-11-14 11:26:49.821 - INFO - Pass ✓... (test-add-variable): add pass
2021-11-14 11:26:49.825 - ERROR - Fail ✗... (test-add-variable): add fail 
                                    Expected: 3
                                    Got: 2
2021-11-14 11:26:49.835 - INFO - Pass ✓... (test-sub-variable): sub pass
2021-11-14 11:26:49.839 - ERROR - Fail ✗... (test-sub-variable): sub fail 
                                    Expected: 4
                                    Got: 0
